svc-aggregation/README.md

```diff
@@ -488,7 +621,7 @@ curl -i GET \
 
 ```
 
-> Sample response body
+>**Sample response body**
 
 ```
 {
```

Comment:
should be used connection method, not oem

Reply:
Changed it.